atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/Schema.java

```diff
@@ -389,10 +403,11 @@ private void emit(File srcDir, String code, String packName, String className)
     }
 
     public void addCleanupTask(String rawTableName, OnCleanupTask task) {
```

Comment:
Maybe add a comment to document this is used by our large internal product

Reply:
Comment added!